tests/src/clarity-tests.ts

```diff
@@ -126,6 +130,18 @@ describe('Clarity Types', () => {
       });
       expect(Object.keys(serializedDeserialized.data)).toEqual(lexicographic);
     });
+
+    test('StringAsciiCV', () => {
+      const str = stringAsciiCV('hello world');
```

Comment:
While working on this feature, clarity side, we uncovered a bug in the parser involving escaped characters. Can we have some testing around this, with strings involving characters from this list 
https://github.com/blockstack/stacks-blockchain/blob/b168a9df92c6deeaccd80bbdbb1b403ea8cbf48c/src/vm/ast/parser/mod.rs#L343-L351

and make sure that theses are correctly escaped (encoded with 1 byte, not 2 bytes)